tests/Struct/FindologicProductTest.php

```diff
@@ -1367,6 +1480,23 @@ public function testProductListPrice(?string $currencyId, bool $isPriceAvailable
         $this->assertSame($isPriceAvailable, $hasListPriceNet);
     }
 
+    private function filterThumbnails(MediaThumbnailCollection $thumbnails): MediaThumbnailCollection
```

Comment:
See my other comment. This is not how tests are done.

Reply:
Changing this method is unnecessary work in my opinion, as is it just filtering and sorting an array. If you are worried about the correctness of the implementation then that is justified - but we can add a simple assertion to make sure it filters how we expect it.

Reply:
I would understand sorting, but filtering shouldn't be done here. In your test you must know your expectations before you start a test. If you just use the same logic, as in the implementation, you aren't really ensuring that the order or the filtering is right.

I would accept this helper if there was a separate test that ensures that the thumbnails are properly filtered and sorted, where you already know the outcome of the filtering and sorting.